proxy/src/coco/git_helper.rs

```diff
@@ -0,0 +1,46 @@
+//! git-remote-rad git helper related functionality.
+
+use crate::config;
+
+use crate::error;
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
```

Comment:
```suggestion
use std::fs;
use std::os::unix::fs::PermissionsExt;

use crate::config;
use crate::error;
```

We order `std -> third party crates -> radicle crates -> workspace crates -> crate modules`

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/769/commits/879708bcd37f25d6839658f1aeec2fec61992cb3